pick.lic

```diff
@@ -79,12 +79,17 @@ class LockPicker
         next
       end
 
-      until 'not even locked' == bput("pick my #{box} blind", 'not even locked', 'Roundtime')
```

Comment:
`until` is a loop construct. By removing it, it appears that this code will no longer loop. Is that intentional?